Sources/Helpers/OAAppSettings.h

```diff
@@ -152,9 +152,94 @@ typedef NS_ENUM(NSInteger, EOAAutoZoomMap)
 
 @end
 
-@interface OAProfileSetting : NSObject
+typedef NS_ENUM(NSInteger, EOAWikiArticleShowConstant)
+{
+    ON = 0,
```

Comment:
Since obj c enums do not have namespaces, it is generally a good idea to avoid such generic names for public enums. Prefer using ```EOAWikiArticleShowConstantOn``` format in such cases